docs/_getting-started/index.md

```diff
@@ -121,56 +135,62 @@ web3.fromWei(eth.getBalance("<ADDRESS_1>"),"ether")
 Getting the balance of an account does not require a signed transaction,
 so Clef does not ask for approval, and Geth returns the value.
 
+Note that this step requires the initial synchronization to end. If you get an error message, return to the 
+console with the network node Geth and wait until it is synchronized. You
+know that your Geth is synchronized when it is only importing a small number of blocks (one or two, usually)
+at a time.
+
 ### Send ETH to account
 
 Send 0.01 ETH from the account that you added ETH to with the Görli faucet,
 to the second account you created:
 
 ```javascript
-eth.sendTransaction({from:"<ADDRESS_0>",to:"<ADDRESS_1>", value: web3.toWei(0.01,"ether")})
+eth.sendTransaction({from:"<ADDRESS_1>",to:"<ADDRESS_2>", value: web3.toWei(0.01,"ether")})
 ```
 
-This action does require signing the transaction, so Clef prompts you to approve it, and
-if you do, asks you for the password you are sending the ETH from. If the password is
-correct, Geth proceeds with the transaction.
+This action does require signing the transaction, so go to the command line window with Clef running 
+to see that Clef prompts you to approve it, and when you do, asks you for the password for the account you are 
+sending the ETH from. If the password is correct, Geth proceeds with the transaction.
 
 To check, get the account balance of the second account:
 
 ```javascript
-web3.fromWei(eth.getBalance("<ADDRESS_1>"),"ether")
+web3.fromWei(eth.getBalance("<ADDRESS_2>"),"ether")
 ```
 
 ## Using RPC
 
 ### Connect to RPC
 
-You can use standard HTTP requests to connect to a Geth node using the RPC APIs, for example:
+You can use standard HTTP requests to connect to a Geth node using the RPC APIs, using
+this syntax:
 
 ```shell
 curl -X POST http://<GETH_IP_ADDRESS>:8545 \
-    -H "Content-Type: application/json" \
-   --data'{"jsonrpc":"2.0", "method":"<API_METHOD>", "params":[], "id":1}'
+    -H "Content-Type: application/json" \
+   --data'{"jsonrpc":"2.0", "method":"<API_METHOD>", "params":[], "id":1}'
 ```
 
 ### Check account balance
 
 ```shell
 curl -X POST http://<GETH_IP_ADDRESS>:8545 \
-    -H "Content-Type: application/json" \
-   --data '{"jsonrpc":"2.0", "method":"eth_getBalance", "params":["<ADDRESS_1>","latest"], "id":1}'
+    -H "Content-Type: application/json" \
+   --data '{"jsonrpc":"2.0", "method":"eth_getBalance", "params":["<ADDRESS_1>","latest"], "id":1}'
 ```
 
-Getting the balance of an account does not require a signed transaction,
-so Clef does not ask for approval, and Geth returns the value.
+Getting the balance of an account does not require a signed transaction, so Clef does not ask for approval, and Geth returns the value.
```

Comment:
(note: clef isn't even invoked here, because there's nothing geth needs from clef in this case)